resource/writers/match_writers.cpp

```diff
@@ -400,6 +453,8 @@ json_t *jgf_match_writers_t::emit_vtx_base (const f_resource_graph_t &g,
                                             unsigned int needs, bool exclusive)
 {
     json_t *o = NULL;
+    if (g[u].type == "node")
+        nodename =  g[u].basename;
```

Comment:
Fluxion designates member variables with `m_`:
```suggestion
        m_nodename = g[u].basename;
```